<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="MAIN_PAGE">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>701</width>
        <height>141</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <hintingpreference>PreferDefaultHinting</hintingpreference>
       </font>
      </property>
      <property name="text">
       <string>XML EDITOR AND NETWORK ANALYSIS</string>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="BROWSE_BUTTON">
      <property name="geometry">
       <rect>
        <x>482</x>
        <y>370</y>
        <width>151</width>
        <height>75</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>BROWSE FILE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="COPY_BUTTON">
      <property name="geometry">
       <rect>
        <x>166</x>
        <y>370</y>
        <width>150</width>
        <height>75</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>COPY FILE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>500</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>incase of decompressing kindly click Browse</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="COPY_PAGE">
     <widget class="QLabel" name="COPY_LABEL">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>600</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PLEASE PASTE TEXT FILE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="XML_TEXT">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>450</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="OK_COPY_BUTTON">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>290</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CLOSE_COPY_LABEL">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>380</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="XML_PAGE">
     <widget class="QLabel" name="features">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>250</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Features </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="please_choose">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>please choose your desired feature</string>
      </property>
     </widget>
     <widget class="QComboBox" name="dropdown">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Choose the feature</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Check Consistency</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prettifying</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>to JSON</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Minifying</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Compress</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Decompress</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Network Analysis</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Graph Visualization</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Search</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="back_button">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>510</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="NETWORK_PAGE">
     <widget class="QLabel" name="Network_Page">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>321</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Network_page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="most_follower">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Most Follower</string>
      </property>
     </widget>
     <widget class="QPushButton" name="most_active">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Most Active</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mutual_followers">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mutual Followers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="suggestions">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>430</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Suggest followers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_button_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>530</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Search">
     <widget class="QLabel" name="Search_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>380</y>
        <width>231</width>
        <height>85</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchByTopic">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Search by topic</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchByWord">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>280</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Search by word</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchBack">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>520</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
